src/containers/profile/ProfileSections/AccountSettings/AccountSettingsSections/ProfileInfo/ProfileInfo.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { connect } from "react-redux";
+import "./ProfileInfo.css";
+import profileInfoEditButtonIcon from "../../../../../../images/editButton.png";
+import ShowProfileInfo from "./ProfileInfoSections/ShowProfileInfo";
+import EditProfileInfo from "./ProfileInfoSections/EditProfileInfo";
+import { Modal, Button } from "react-bootstrap";
+
+const UserInfo = (props) => {
+  const { id, authError, errMessage, errKey } = props;
+  const [isEditing, setIsEditing] = React.useState(false);
+  const [
+    profileInfoEditButtonOpacity,
```

Comment:
this is fine for now, but in the future please do not use magic strings like "opacityLow" as state values. instead optimally this would be a boolean value and on line 23 you would add a class if the boolean value is true (the boolean would be called like `isProfileButtonLowOpacity`